components/Button.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+const Button = ({ type, children, onClick, size, width, height, padding, margin }) => (
+  <button type={type} onClick={onClick} className="button-root">
+    {children}
+    <style jsx>{`
+      .button-root {
+        display: block;
```

Comment:
A button should not always be `block` and the default works better which is `inline-block` :)

Reply:
for my current use case mostly its block level, but i did add an optional prop for display